### PR TITLE
The dual esword can no longer be constructed

### PR DESCRIPTION
**Pull request by Cheridan**
:cl:
del: Dualsabers are no longer constructable
/:cl:

### CODE REVIEW

**Comment by Jazaen** [comment]
>Thumbing up your own PR

**Comment by InsaneHyena** [comment]
> Thumbing up your own PR

**Comment by F-OS** [comment]
HELLO I AM Cheridan AND I MAKE PRS WHEN I GET MURDERBONED AND MY ARRPEE IS INTERRUPTED!

**Comment by F-OS** [comment]
> Thumbing up your own PR

**Comment by Iamgoofball** [comment]
:+1:

**Comment by Thunder12345** [comment]
Lrn2play

**Comment by PKPenguin321** [comment]
:-1: They are not even close to as much of an issue as people claim they are

**Comment by Ergovisavi** [comment]
I'm pretty disappointed in this kind of reaction, especially from someone with the headcoder position.

Cmon, Cheridan. You're better than this.

**Comment by Cheridan** [comment]
Ergo, this is exactly the reaction I need to take. Ultimately, dualsabers do not contribute to the style of play that SS13 should be focused around. It is a game of paranoia and subterfuge, and dualsabers run counter to that.

**Comment by InsaneHyena** [comment]
If you want the paranoia, how about we remove one thousand ways to get back into the round, increase the time that must pass before the shuttle can be called, remove gang and cockcult? Oh wait, none of this is gonna happen.

**Comment by Supermichael777** [comment]
it hasn't been that game for a long time.
Im not saying that this is a bad thing
but if your going to die on this hill you might as well make it bigger

**Comment by Cheridan** [comment]
Gotta start somewhere, Michael.

**Comment by Thunder12345** [comment]
Hint: 90% of the time dualsabers are nothing more than a shiny, expensive trap to kill murderboners. There's plenty of easily accessible counters to them. Shotguns, the detective, spears, tiles, bear traps, and simply overwhelming them with a wave of angry greyshirts.

**Comment by Limed00d** [comment]
This proper disgusts me.

**Comment by Limed00d** [comment]
I find this PR particularly funny because last I checked you guys still had this massive gap between coders and players.

**Comment by Iamgoofball** [comment]
@Cheridan this IS a deathmatch action game

**Comment by PKPenguin321** [comment]
>Gotta start somewhere, Michael.

Unless you plan on continuing to make PRs like this once this one is done, then this comment is absolute bullshit

**Comment by Thunder12345** [comment]
SYBIL: http://puu.sh/qZSvw/a1385722f4.png

BASIL: http://puu.sh/qZSrA/69380b05b2.png

**Comment by Cheridan** [comment]
@PKPenguin321 Depends on what you mean by "like this one." I plan on making more prs to move the game away from deathmatch grieflottery, yes. But that was always the case.

**Comment by Iamgoofball** [comment]
@Thunder12345 incorrect usage of player polling

**Comment by Thunder12345** [comment]
@Iamgoofball How so?

**Comment by Iamgoofball** [comment]
you only grabbed feedback from the active players at the moment

ignoring all other timezones and players not currently connected

run a proper DB poll for a week to get an actual set of data

**Comment by Bluespace** [comment]
you don't need to run a poll.
anyone who understands and cares about what ss13 actually is isn't in favor of removing them.
ss13 is based around conflict, but it is also based around dealing with that conflict and reversing it, that is the game. 
remove dual eswords if you want, but also make the singularity start working, atmos be ai proof, chemistry start stocked, etc etc. 
tl;dr: you're supposed to die and then be brought back to life in ss13.

**Comment by lzimann** [comment]
I think people need to get good instead of removing stuff like this

**Comment by Jalleo** [comment]
Either way this goes it doesnt actually effect the game a big deal. Everyone is arguing over a mole hill. The actual issue is SS13 combat and 2D. Lets go 3D and make combat really crazy again.

**Comment by optimumtact** [comment]
Some of the shitposting was cleaned.

**Comment by Isratosh** [comment]
Keep the dual saber

**Comment by Kromgar** [comment]
BOOO HISSSSS

**Comment by Fox-McCloud** [comment]
Really not a fan of this, to be honest. The dual saber is strong, but I'm not really sure it's the be all/end-all its made out to be....and certainly its something that could be nerfed, first, before it's out and out removed.

There's a lot of levers you can pull to change how effective it is, especially with @KorPhaeron refactor of how shields work. 

You could lower its block chance, make its reflect chance probability based (or removed), lower the damage, etc.

That said, the item is what? 16 Telecrystals? It should be strongly. Someone with a revolver and autolathe can pretty much just run around two-shotting everyone, and that's from range.

**Comment by Cheridan** [comment]
@lzimann there's nothing to get good at, you either have the handful equipment available that can actually give you a chance against it or you don't. "Getting gud" at SS13 means making bolas and stunprods at roundstart.

@bluespace 
>anyone who understands and cares about what ss13 actually is isn't in favor of removing them.

A murderbone simulator where you try to get the best equipment so that you can kill as many people as you can? Because that's what Sibyl is turning into. 

>remove dual eswords if you want, but also make the singularity start working, atmos be ai proof, chemistry start stocked, etc etc. 

Except having a stocked chemistry and a working roundstart singularity doesn't actually have anything to do with balance? It's just the 5 minute redundant preptime that we expect people to do every 30 minutes? It certainly doesn't have any impact on how badly designed this weapon is.

**Comment by lzimann** [comment]
Lower the melee block chance, there you go, that should actually solve problems instead of completly removing a fun thing.

**Comment by Fox-McCloud** [comment]
I don't personally think that's a good argument @Cheridan. And not because "improve don't remove! HUR DUR", but because invariably, if it's not this that's "powercreeped", it'll be something else. Yes, people *could* rebuff it back to where it was, but they could also re-add it, as well.

If the weapon is a legitimate problem (notttt so sure about that myself), then it makes more sense to try to balance it, first, rather than destroying it, especially when, again, you do have so many levers to pull.

Either case, my 2 cents, so to speak.

**Comment by WJohn** [comment]
I think we should just make it so you can't lop off limbs from people who are standing, as that's where the OHK nature of it comes from. If you aim for the limbs of an unarmored person (and seemingly any armored person, too?) You are guaranteed to cut off their leg in one hit -- which is a knockdown -- which is followed by death. This is like a stunbatton that deals 35 brute per hit, reflects all energy/laser projectiles, and requires no charge. Yeah eventually someone might get wise and use the detective's revolver on your or get a lucky melee strike on your with a stun baton/disarm, but by then you've probably killed 15-20 people.

**Comment by KorPhaeron** [comment]
@Cheridan 

Should add them to the ops uplink at least, they fit well with that mode

**Comment by Nanotransistor** [comment]
Why not start by removing the game modes that represent nothing of what SS13 is about, like gangs, all the cults and revs?

**Comment by pubby** [comment]
I'd +1 this commit if it were followed by "Add duel eswords to uplinks for 18 TC". The constructible aspect really only served to be a roadblock to balance; duel esword costs couldn't be changed without also affecting single esword costs.

**Comment by ike709** [comment]
>A murderbone simulator where you try to get the best equipment so that you can kill as many people as you can? Because that's what Sibyl is turning into.

Try Basil then.

**Comment by Cheridan** [comment]
@KorPhaeron Fair enough, I'll do that.

@pubby They're already 18 though, the only place you can really go from there is 20 tc to prevent buying no-slips with them. Which would help, but it's still an item that by design locks you into murderbone mode where you now exist just to kill as many people as possible.

@ike709 No.

@Nanotransistor SS13 game modes have always represented small departures from the core Traitor gameplay. I'm fine with Nuke Ops and Wizard being fast-paced action modes, or the strategic teamwork of Clockcult. This item doesn't mesh with what Traitor is supposed to be about, though.

**Comment by lzimann** [comment]
They are 16 right now(8 each sword)

**Comment by Cheridan** [comment]
They're 18 total with the obligatory noslips. So we'd have to make them 19 or 20 TC.

**Comment by NocturnalQuill** [comment]
i ded pls nerf

**Comment by AdamElTablawy** [comment]
I'd say that them being ops-only is logical.

Or if you could somehow tie them to objectives, hijack traitors could also be able to buy dual eswords.

Removing the ability to de-limb people as well and make the blockchance not as insane.

**Comment by NocturnalQuill** [comment]
I guess I would be okay with increasing the cost of eswords to make them 9 TC each to account for the new functionality that dismemberment brings. Removing it altogether though is fucking stupid and you should feel stupid.

**Comment by Firecage** [comment]
Bad PR with no solid foundation.

**Comment by lzimann** [comment]
Maybe removing the dismember is enough, it started being waaay more used when i fixed them not dismembering

**Comment by Cheridan** [comment]
Even if it's nerfed, it's role in the game is still just to be a mass-murderbone weapon. The only nerf that'd make it suitable for Traitor would be to put it back where it was (on par with normal eswords, so it regains its place as an easter egg/show-off item).

**Comment by Cheridan** [comment]
Anyway nobody's actually convinced me that these things have any place in the game but I don't want everyone butthurt at me so whatever :last_quarter_moon:

**Comment by Ninja137** [comment]
It's alright to be a fag, Cher.

If you're going to try and fix antag items, why don't you try, I don't know, actually adding interesting new items for traitors for almost the first time in years? Stetchkin is ass, revolver is a meme, Ebow is KA+, and melee is melee.

**Comment by blobbernaut** [comment]
What a fucking childish response lmao

**Comment by Cheridan** [comment]
kor told me to

**Comment by Cheridan** [comment]
oops oranges said i need to close this and make one without 30 downvotes on it already

**Comment by blobbernaut** [comment]
Fuck off kor you salty fuck

**Comment by Aranclanos** [comment]
i fully support this change everyone else is prob bad lol just git gud

**Comment by F-OS** [comment]
this is the most downvoted PR in history so :+1:
